docs/geps/gep-01.rst

```diff
@@ -0,0 +1,279 @@
+Each of these parameters then has at least three keys: ``name``, ``description``, and
+``value``. Values usually change over time; each time a value is introduced or changed
+the respective law or regulation is referenced along with the numerical value.
+
+1. The ``name`` key has two sub-keys ``de`` and ``en``, which are
+
+   * short names without stating the realm (e.g. "Arbeitslosengeld II" or
+     "Kinderzuschlag") again,
+   * not sentences,
+   * correctly capitalised.
+
+   Example (from ``arbeitsl_geld_2``):
+
+   .. code-block:: yaml
+
+        name:
+          de: Regelsatz
+          en: Standard rate
+
+2. The ``description`` key has two sub-keys ``de`` and ``en``, which
+
+   * are good and full explanations of the parameter
+   * show the § and Gesetzbuch/Paragraph (history) of that parameter
+   * mention bigger amendments/Neufassungen and be as helpful as possible to
+     make sense of that parameter
+
+    Example:
+
+   .. code-block:: yaml
+
+        description:
+          de: Einkommensanteil, der anrechnungsfrei bleibt, Intervall 2 [a2eg1, a2eg2]. § 30 SGB II. Seit 01.04.2011 § 11b SGB II.
+          en: Income share not subject to transfer withdrawal, interval 2 [a2eg1, a2eg2]. § 30 SGB II. Since 01.04.2011 § 11b SGB II.
+
+
+3. The ``value`` key
+
+   * contains the value as defined in the law
+   * values in percentages can alternatively be expressed to the base of one
+   * add a leading zero for values smaller than 1 and greater than -1
+   * DM values have to be converted to Euro using the excange rate 1:1.95583.
+
+   Example 1:
+
+   .. code-block:: yaml
+
+        value:
+            2005-01-01:
+                value: 1500
+                note: Artikel 1 G. v. 24.12.2003 BGBl. I S. 2954.
+            2005-10-01:
+                value: 1200
+                note: Artikel 1 G. v. 14.08.2005 BGBl. I S. 2407.
+
+   Example 2:
+
+   .. code-block:: yaml
+
+        values:
+            1995-01-01:
+              value: 681
+              note: Artikel 31 G. v. 23.06.1993 BGBl. I S. 944. Wert in Euro. Der Betrag im Gesetz lautet 1332 DM.
+            1998-01-01:
+              value: 939
+              note: Artikel 1 G. v. 21.11.1997 BGBl. I S. 2743. Wert in Euro. Der Betrag im Gesetz lautet 1836 DM.
+            2002-01-01:
+              value: 972
+              note: B. v. 15.10.2002 BGBl. I S. 4130.
+            2021-01-01:
+              value: 16956
+              note: Artikel 1 G. v. 10.12.2019 BGBl. I S. 2115.
+
+
+   the `note` key:
+
+   * cites the law (Gesetz "G. v."), decree (Verordnung "V. v.") or proclamation (Bekanntmachung "B. v.") that changes the law
+   * in German
+   * follows the style ``Artikel [n] [G./V./B.] v. [DD.MM.YYYY] BGBl. I S. [SSSS]. [optional note].``
+   * do not add information "geändert durch" (it is always a change) or the date the law comes into force (this is exactly the date key in the previous line)
```

Comment:
the last parantheses is not correct. The date in the note is the date of publishing. The date of coming into force is below the `values` key.

Reply:
I think that is precisely what @mjbloemer wrote. Am I missing something? Do you have a better suggestion less prone to misunderstandings?

Feel free to edit directly, also regarding the two comments below, it is getting kind of hard to follow here.